README.md

```diff
@@ -100,61 +115,91 @@ List of references used for each project on the Portal with `CellAssign`, includ
 
+## Additional repository contents
 
-## Sample info
+### Sample info
 
-The `sample-info` folder contains metadata files used to create figures and tables.
+The `sample-info/` folder contains metadata files used to create figures and tables.
 
-1. `diagnosis-groupings.tsv`: This tsv file contains one row per `submitted_diagnosis` associated with samples on the ScPCA Portal.
+* `brain-classifications-no-multiplexed.tsv`: This tsv file classifies brain-related diagnoses in the ScPCA Portal into "High-grade glioma" and "Low-grade glioma" for plotting.
+* `celltype-reference-metadata.tsv`: This tsv file contains information about references used for CellAssign cell type annotation on the ScPCA Portal.
+* `diagnosis-groupings.tsv`: This tsv file contains one row per `submitted_diagnosis` associated with samples on the ScPCA Portal.
 For each `submitted_diagnosis`, a `diagnosis_group` is assigned.
-
-2. `disease-timing.tsv`: This tsv file contains one row per `submitted_disease_timing` associated with samples on the ScPCA Portal.
+* `disease-timing.tsv`: This tsv file contains one row per `submitted_disease_timing` associated with samples on the ScPCA Portal.
 For each `submitted_disease_timing`, a `standardized_disease_timing` is assigned.
+* `project-whitelist.txt`: This file contains a list of all projects that are currently active on the ScPCA Portal.
+* `sample-whitelist.txt`: This file contains a list of all samples that are currently active on the ScPCA Portal.
+* `scpca-project-celltype-metadata.tsv`: This tsv file provides the specific CellAssign cell typing reference used for each ScPCA project.
 
-3. `project-whiteliest.txt`: This file contains a list of all projects that are currently active on the ScPCA Portal.
 
-## Color palettes
+### Color palettes
 
-The `palettes` folder contains any palettes used in generating the figures.
+The `palettes/` folder contains any palettes used in generating the figures.
 
-1. `diagnosis-group-palette.tsv`: This is the palette used to color the `diagnosis_group` for each sample.
-2. `suspension-palette.tsv`: This is the palette used to color libraries by `Single-cell` or `Single-nuclei`.
-3. `method-palette.tsv`: This is the palette used to color by quantification method used, either `Alevin-fry` or `Cell Ranger`.
+* `diagnosis-group-palette.tsv`: This is the palette used to color the `diagnosis_group` for each sample.
+* `disease-timing-palette.tsv`: This is the palette used to color the `disease_timing` for each sample.
+* `immune-palette.tsv`: This is the palette used to color certain immune cell types from the overall consensus cell types.
+* `suspension-palette.tsv`: This is the palette used to color libraries by `Single-cell` or `Single-nuclei`.
+* `method-palette.tsv`: This is the palette used to color by quantification method used, either `Alevin-fry` or `Cell Ranger`.
+* `validation-group-palette.tsv`: This is the palette used to color validation groups used to assess consensus cell types.
```

Comment:
```suggestion
* `validation-group-palette.tsv`: This is the palette used to color broad cell type annotations used to assess consensus cell types.
```
Just because we don't really output anything that says validation group, we should be clear what colors in the figures this matches up to.